ci/testgrid/config.yaml

```diff
@@ -49,6 +49,8 @@ default_dashboard_tab:
   num_columns_recent: 10
   code_search_url_template:      # The URL template to visit when searching for changelists
     url: https://github.com/knative/serving/compare/<start-custom-0>...<end-custom-0>
+  alert_options:
+    alert_mail_to_addresses: 'knative-test+testgrid-alerts@googlegroups.com'
```

Comment:
can we send alert to knative-testinfradev@googlegroups.com? 
I can work with Mark to get such group created, and we will use it for all purpose test infra alerts.

Reply:
Done. Holding until the new group is created. I deleted the other group.

/hold